Mathlib/CategoryTheory/Monoidal/Braided.lean

```diff
@@ -123,6 +128,30 @@ theorem braiding_naturality {X X' Y Y' : C} (f : X ⟶ Y) (g : X' ⟶ Y') :
   rw [tensorHom_def' f g, tensorHom_def g f]
   simp_rw [Category.assoc, braiding_naturality_left, braiding_naturality_right_assoc]
 
+theorem yang_baxter (X Y Z : C) :
+    (α_ X Y Z).inv ≫ ((β_ X Y).hom ▷ Z) ≫ (α_ Y X Z).hom ≫
```

Comment:
We don't need parentheses around whiskerings.

Reply:
I dislike writing a series of infix operators when it's unclear as a reader what the precedence is, even if it's unambiguous to lean

Reply:
Much of the mathlib code seems to use as few parentheses as possible (not all, of course). The priority is not obvious from the text, but we can see it by hovering over it in VSCode. I think this is one of the reasons why we don't need parentheses in practice.